airbyte-commons/src/main/java/io/airbyte/commons/lang/MoreBooleans.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2020 Airbyte
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package io.airbyte.commons.lang;
+
+public class MoreBooleans {
+
+  /**
+   * Safely handles converting boxed Booleans to booleans, even when they are null. Evaluates null as
+   * false.
+   *
+   * @param bool boxed
+   * @return unboxed
+   */
+  public static boolean evaluateNullAsFalse(Boolean bool) {
```

Comment:
open to other names here. the value of this is proportional to the brevity of the name and this name is not that brief.

Reply:
`isFalsy` this way it is brief and you can always go the comment to understand the handling of null. Similar to Guava `Strings.isEmpty` & `Strings.isBlank`

Reply:
would do what michel said but as `isTruthy` instead so we don't have to do mental hoops